templates/vue-demo-store/pages/checkout/index.vue

```diff
@@ -204,12 +209,20 @@ const placeOrder = async () => {
     termsBox.value?.scrollIntoView();
     return;
   }
+  if (!beforeCreateOrderValidation()) return;
 
   isLoading["placeOrder"] = true;
-  const order = await createOrder();
-  isLoading["placeOrder"] = false;
-  await push("/checkout/success/" + order.id);
-  refreshCart();
+
+  try {
+    const order = await createOrder();
+    console.log("order", order);
+    isLoading["placeOrder"] = false;
```

Comment:
this should go to the `finally` block as in case of error loading will be always `off`